Lib/ufo2ft/__init__.py

```diff
@@ -738,6 +753,18 @@ def compileVariableCFF2s(designSpaceDoc, **kwargs):
     return vfNameToTTFont
 
 
+def _featuresCompatible(designSpaceDoc):
+    def transform(f):
+        # Strip comments
+        text = re.sub("(?m)#.*$", "", f.font.features.text or "")
+        # Strip extraneous whitespace
+        text = re.sub(r"\s+", " ", text)
+        return text
+
+    first = transform(designSpaceDoc.sources[0])
+    return all(transform(s) == first for s in designSpaceDoc.sources[1:])
```

Comment:
At this stage, features are not expanded to masters, so differing imports are not checked. Is this a problem?

Reply:
Not sure I understand this. What does expanding features to masters mean?

Reply:
Resolving all imports with `parseLayoutFeatures`. We can't test for equality what we can't get our hands on. I suppose that this is an edge case, if you use feature file syntax without variation conditionals, checking just the master feature text should be enough, because you can't change master behavior elsewhere without it being imported everywhere else.

Reply:
If the files are identical, any imports will import the same files. So that case is OK. If the UFOs import different feature files which turn out to be the same code, then the font engineer is pathological and deserves to pay a small performance penalty.

Reply:
Never discount the possibility of a small bug somewhere going unnoticed :)